app/views/admin/users/index/_member_data.html.erb

```diff
@@ -1,8 +1,8 @@
-<a href="<%= admin_user_path(user.id) %>" class="mr-2">
+<a href="<%= admin_user_path(user.id) %>" class="mr-2 c-link">
     <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
 </a>
 <div class="flex flex-col">
-    <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
+    <h3 class="fs-base"><%= link_to user.name, admin_user_path(user), class: "c-link" %></h3>
```

Comment:
Thank you for adding the `c-link`s! It seems like there was a regression between the initial data-filled PR and this one that removed the class that was on the username (although, the classes that you and I used were different!)--when I pulled latest the other day I wondered if it was intentional or not. This looks so much better! 😄

Reply:
Yeah I think this must have been one of my PRs crossing with yours and wiping something out 😂